Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN	dpkg --add-architecture i386 && \
 	apt-get install -y --no-install-recommends xvfb x11vnc xdotool supervisor fluxbox xterm net-tools nodejs && \
 # Adding required ppas: for installing wine.
 	apt-get purge wine.* && \
-	wget -nc https://dl.winehq.org/wine-builds/Release.key && apt-key add Release.key && \
+	wget -nc https://dl.winehq.org/wine-builds/winehq.key && apt-key add winehq.key	&& \

```

Comment:
Necessary because repository key has changed, see https://wiki.winehq.org/Ubuntu

Reply:
Thank you for the contribution, the changes looks good.